docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.md

```diff
@@ -309,9 +309,7 @@ test('My test', async t => {
 });
 ```
 
-## More Examples
+## Have a different use case?
 
-If you encounter a difficult situation while working with DOM elements,
-let us know by posting on StackOverflow. We review and answer questions
-with the [TestCafe](https://stackoverflow.com/questions/tagged/testcafe) tag.
-If you are not alone we will add an example to this topic.
\ No newline at end of file
+If none of the examples fit your requirements and you encounter any difficulties, let us know on StackOverflow.
```

Comment:
```suggestion
If none of the examples fit your requirements and you encounter difficulties, contact DevExpress on StackOverflow.
```

Reply:
> Contact DevExpress
may not be a suitable option in this case, as users cannot 'contact' us there, per se, as it is not really our service. StackOverflow can be used by anyone for any questions, the only option to connect with us is just let us know by mentioning our tag, which is also included.

Reply:
```suggestion
If none of the examples fit your requirements and you encounter difficulties, let us know on StackOverflow.
```